functions-methods.js

```diff
@@ -33,4 +57,32 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(emailAddress1) {
+    const dot = emailAddress1.includes(".");
+    const comma = emailAddress1.includes(",");
+    const at = emailAddress1.includes("@");
+
+
+
+    if(dot > at &&  emailAddress1.length) {
+       console.log("True")
+   }
+    if(comma === true) {
```

Comment:
Als je zulke expressies gaat maken, dan kun je dit ook korter schrijven, omdat `comma` op zichzelf al een boolean is:
`if (comma) { ... }`